scripts/promote_docker_image_and_chart.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+drone_cli_help()
+{
+    echo "  Requires Drone CLI and personal access token"
+    echo "  Download CLI following instructions at https://docs.drone.io/cli/install/"
+    echo "  Then configure access token: https://docs.drone.io/cli/setup/"
+    echo ""
+    echo "  To test if Drone CLI is properly configured type:"
+    echo ""
+    echo "     drone build ls rancher/rancher"
+    echo ""
+    echo "  This will show the last 25 builds"
+}
+
+if [[ $# -ne 2 ]]; then
+    echo "Promote Chart and Docker image to stable or latest tag"
```

Comment:
There is no action to promote a chart to latest as that's part of the drone process to tagging a release itself. It goes directly into the Helm repo for "latest". 

So, I think the options are to take the 2 existing scripts that are:
1. Promote a Docker image to stable or latest
2. Promote a Chart to stable

And update them to become:
1. Promote a Docker image to latest
2. Promote a version to stable, which is promoting a docker Image to stable tag and the Helm chart into the stable Helm repo